br/metrics/grafana/lightning.json

```diff
@@ -39,6 +39,12 @@
       "id": "table",
       "name": "Table",
       "version": ""
+    },
+    {
+      "type": "panel",
+      "id": "bargauge",
```

Comment:
seems this type of panel is introduced since v6.2 https://hub.packtpub.com/grafana-6-2-released-with-improved-security-enhanced-provisioning-bar-gauge-panel-lazy-loading-and-more/

I'm not sure which version do we use in deployment

Reply:
In cloud dev clinic is using v9.4.2 which is okay. I have tested tiup cluster and it also supports bar gauge so I think it's fine.